app/routes.py

```diff
@@ -73,3 +74,18 @@ async def change_password_route(
 ):
     account = await authenticate_from_token(access_token)
     return await change_password(account, old_password, new_password)
+
+
+@app.post('/forgot-password')
```

Comment:
Our hero ![Florimond](https://florimond.dev/blog/articles/2018/08/restful-api-design-13-best-practices-to-make-your-users-happy/) recommends not using verbs as REST routes (same comment for most of the routes)

Reply:
This backend is not restful 😅😅 Maybe I will make it more REST compliant some time.